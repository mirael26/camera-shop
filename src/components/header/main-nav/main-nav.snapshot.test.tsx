import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import MainNav from './main-nav';

test('MainNav renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<MainNav/>);
  expect(asFragment()).toMatchSnapshot();
});
