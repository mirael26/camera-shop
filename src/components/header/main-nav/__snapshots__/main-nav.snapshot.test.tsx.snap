// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav renders correctly 1`] = `
<DocumentFragment>
  <nav
    class="main-nav"
    data-testid="main-nav"
  >
    <ul
      class="main-nav__list"
    >
      <li
        class="main-nav__item"
      >
        <a
          class="main-nav__link"
          href="/catalog"
        >
          Каталог
        </a>
      </li>
      <li
        class="main-nav__item"
      >
        <a
          class="main-nav__link"
          href="example.com"
        >
          Гарантии
        </a>
      </li>
      <li
        class="main-nav__item"
      >
        <a
          class="main-nav__link"
          href="example.com"
        >
          Доставка
        </a>
      </li>
      <li
        class="main-nav__item"
      >
        <a
          class="main-nav__link"
          href="example.com"
        >
          О компании
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
